devtools/conda-envs/test_env.yaml

```diff
@@ -32,7 +32,11 @@ dependencies:
   - qcelemental=0.25.1
   - qcportal>=0.50
   - requests
+  - jax
+  - flax
+  - versioneer
 
   # pip installs
   - pip:
       - schnetpack>=2.0.0
+      - git+https://github.com/ArnNag/sake.git@update_versioneer
```

Comment:
I am not sure what this installation is doing?

Reply:
This is also for the correctness tests